core/cmd/app.go

```diff
@@ -468,6 +468,25 @@ func NewApp(client *Client) *cli.App {
 							},
 							Action: client.ExportETHKey,
 						},
+						{
+							Name:   "reset",
+							Usage:  "Manually set the next nonce for a key. This should not be necessary during normal operation. USE WITH CAUTION: Setting this incorrectly can break your node",
+							Action: client.ResetETHKey,
+							Flags: []cli.Flag{
+								cli.StringFlag{
+									Name:  "address",
+									Usage: "address of the key for which to set the nonce",
+								},
+								cli.StringFlag{
+									Name:  "evmChainID",
+									Usage: "key nonce will be reset for this chain ID only",
+								},
+								cli.Uint64Flag{
+									Name:  "nextNonce",
+									Usage: "the next nonce in the sequence",
```

Comment:
I am thinking to add a fourth optional flag here called `"nuke"` or `"hard"`. This would implement [ch43120] and would also remove all pending/in-flight eth transactions.